src/components/article/mixins/FitwidthMixin.js

```diff
@@ -4,12 +4,14 @@ import _ from 'lodash'
 let FitwidthMixin = (superclass) => class extends superclass {
   componentDidMount() {
     // set state for the width of the images and listen to window resize event
+    if (this._isFoundDOMNode()){
     this.fitToParentWidth()
     window.addEventListener('resize', this.fitToParentWidth)
+    }
```

Comment:
in ```componentDidMount``` function, do we need to check if we are on the client side?
I thought ```[componentDidMount](https://facebook.github.io/react/docs/component-specs.html#mounting-componentdidmount)``` only be invoked on the client side.